src/main/java/de/zalando/zmon/dataservice/config/OpenTracingConfigProperties.java

```diff
@@ -0,0 +1,102 @@
+package de.zalando.zmon.dataservice.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "opentracing")
+public class OpenTracingConfigProperties {
+
+    private String tracingProvider = "jaeger";
+    private String openTracingServiceName = "zmon-data-service";
```

Comment:
Seems like a good default. SpringBoot apps have a property [spring.application.name](https://docs.spring.io/spring-boot/docs/current/reference/html/common-application-properties.html) that returns the application name

Reply:
to use that we have to define that variable in the application.yaml file as follows:
spring:
  application:
    name: abc

I am currently doing it through the environment variables as the other opentracing specific variables

I am wondering which one will be the better way.

Reply:
for me the env variable is the better way...as the user will have control on the open tracing service name